hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,30 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset in state, this field MUST NOT be set.<br/>
+     * If unset in this transaction and set in state, the current value SHALL
+     *  be removed.<br/>
+     * If set in this transaction, the existing key MUST sign this transaction.
+     */
+    Key fee_schedule_key = 10;
+
+    /**
+     * A set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <p>
+     * If this list is empty, the current set of keys is unchanged.
+     */
+    repeated Key free_messages_key_list = 11;
+
+    /**
+     * A set of custom fee definitions.<br/>
+     * These are fees to be assessed for each submit to this topic.
+     * <p>
+     * If this list is empty, the current set of fees is unchanged.
+     */
+    repeated ConsensusCustomFee custom_fees = 12;
```

Comment:
Agree with Michael H, we should use wrappers here

Reply:
Updated to use wrappers